pyrevitlib/pyrevit/__init__.py

```diff
@@ -57,13 +54,8 @@
 except NameError:
     raise Exception('Critical Error. Can not find home directory.')
 
-# Determine dotnet runtime
-DOTNET_RUNTIME_ID = "netfx"
-if System.Environment.Version.Major >= 5 or \
-        (".NET Core" in SRI.RuntimeInformation.FrameworkDescription):
-    DOTNET_RUNTIME_ID = "netcore"
-
-IS_DOTNET_CORE = DOTNET_RUNTIME_ID == "netcore"
```

Comment:
`IS_DOTNET_CORE` is used in pyrevit settings window, so please adjust it accordingly

https://github.com/pyrevitlabs/pyRevit/blob/86697ad59cc66fe73baca529a1670b2c94160506/extensions/pyRevitCore.extension/pyRevit.tab/pyRevit.panel/Settings.smartbutton/script.py#L161

Reply:
@sanzoghenzo I've adjusted